src/DotNetWorker.Core/Hosting/CoreWorkerHostBuilderExtensions.cs

```diff
@@ -38,5 +40,35 @@ public static IHostBuilder ConfigureFunctionsWorker(this IHostBuilder builder, A
 
             return builder;
         }
+
+        /// <summary>
+        /// Invokes auto-generated configuration methods for a given <see cref="IHostBuilder"/>.
+        /// This method searches for classes that implement the <see cref="IAutoConfigureStartup"/> interface,
+        /// excluding interfaces and abstract classes. For each identified class, it locates the
+        /// <c>Configure</c> method with the signature <c>void Configure(IHostBuilder hostBuilder)</c>,
+        /// and executes the method using an instance of the class.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHostBuilder"/> to configure.</param>
+        /// <returns>The same <see cref="IHostBuilder"/> after the auto-generated configuration methods are invoked.</returns>
+        public static IHostBuilder InvokeAutoGeneratedConfigureMethods(this IHostBuilder builder)
```

Comment:
small suggestion: if this is going to be `public`, then consider adding a parameter to accept the `Assembly` that we will scan. This will make it slightly more flexible if devs start calling it directly. Our usage of it can then pass in the entry assembly.

Reply:
Going to leave it as it is for now. If we have a need to use the code in this method in a second place, we shall consider that change.